src/goto-symex/field_sensitivity.cpp

```diff
@@ -0,0 +1,254 @@
+/*******************************************************************\
+
+Module: Field-sensitive SSA
+
+Author: Michael Tautschnig, mt@eecs.qmul.ac.uk
+
+\*******************************************************************/
+
+#include <util/simplify_expr.h>
+#include <util/std_expr.h>
+
+#include "symex_target.h"
+#include "goto_symex_state.h"
+
+#include "field_sensitivity.h"
+
+/*******************************************************************\
+
+Function: field_sensitivityt::operator()
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+void field_sensitivityt::operator()(
+  const namespacet &ns,
+  exprt &expr,
+  bool write) const
+{
+#if 1
```

Comment:
Shall we have command line options to turn on/off field-sensitivity for structs/arrays to make it easier to perform experiments with it?